pkg/entity/kafka_exporter/cmd.go

```diff
@@ -57,8 +57,7 @@ func GetCmd() *cobra.Command {
 			printer = configurePrinter(cmd)
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			recursive, _ := cmd.Flags().GetBool(common.RecursiveFlagName)
-			get(&args[0], recursive)
+			get(&args[0], cmd)
```

Comment:
The `recursive` functionality is removed here.

Reply:
I'll put it back as it is a todo, but it was unused